src/Writers/CSharpWriter/SourceCodeGenerator.cs

```diff
@@ -1,64 +1,58 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
 using System;
```

Comment:
Done. Nope, git picked up this new file as a renamed file. it had a lot of content moved.